test/scripts/e2e-kind.sh

```diff
@@ -58,8 +58,13 @@ should prevent Ingress creation if more than 1 IngressClass marked as default
 
 # TODO: Figure out why the below test is failing and if we need to add support in OVN-K for them
 validates that there is no conflict between pods with same hostPort but different hostIP and protocol
+
+# Skip ServiceCidr tests as we dont currenly support this v1alpha feature
+should create Services and servce on different Service CIDRs
```

Comment:
is `servce` expected :P ? did they also type the tests wrongly upstream in k/k ? :D

Reply:
yes that is the spelling upstream

Reply:
lol